### PR TITLE
Link activites

### PR DESCRIPTION
**Pull request by nvs266**

### CODE REVIEW

File: app/src/main/java/com/framgia/fdms/screen/inventory/InventoryActivity.java [hunk 1]
```diff
@@ -10,26 +10,30 @@
...
 public class InventoryActivity extends AppCompatActivity {
 
+    public static final String EXTRA_USER = "EXTRA_USER";
+
     private InventoryContract.ViewModel mViewModel;
 
     private ActivityInventoryBinding mBinding;
 
-    public static Intent newIntent(Context context) {
+    public static Intent newUpdateHistoryIntent(Context context, User user) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/inventory/InventoryActivity.java]: keep old menthod name

File: app/src/main/java/com/framgia/fdms/screen/inventory/InventoryViewModel.java [hunk 2]
```diff
@@ -70,6 +71,12 @@ public InventoryViewModel(InventoryActivity activity) {
     @Override
     public void onStart() {
         mPresenter.onStart();
+
+        User user = mActivity.getIntent().getParcelableExtra(InventoryActivity.EXTRA_USER);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/inventory/InventoryViewModel.java]: e nên inject user vào viewModel

File: app/src/main/java/com/framgia/fdms/screen/inventory/InventoryActivity.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/screen/inventory/InventoryActivity.java]: (resolved)

File: app/src/main/java/com/framgia/fdms/screen/inventory/InventoryViewModel.java (hunk 2 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/screen/inventory/InventoryViewModel.java]: (resolved)

**Pull request merged**